test/powershell/Modules/Microsoft.PowerShell.Core/Where-Object.Tests.ps1

```diff
@@ -1,5 +1,6 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+. (Join-Path -Path $PSScriptRoot -ChildPath ..\..\Add-TestDynamicType.ps1)
```

Comment:
Please put the help code in BeforeAll block with check that the class is not already added.